packages/manager/apps/web/client/app/hosting/cdn/order/hosting-cdn-order.routing.js

```diff
@@ -104,117 +83,120 @@ export default /* @ngInject */ ($stateProvider) => {
       return null;
     },
 
+    onError: /* @ngInject */ (goBackWithError) => (error) =>
+      goBackWithError(error.data?.message || error),
+
     trackClick: /* @ngInject */ (atInternet) => (hit) => {
       atInternet.trackClick({
         name: hit,
         type: 'action',
       });
     },
   };
+
   const resolveOrder = {
-    prepareCart: /* @ngInject */ (
-      goBackWithError,
-      serviceName,
-      serviceOption,
-      user,
-      $translate,
-      HostingCdnOrderService,
-    ) => () => {
-      const data = { cartId: null };
-      return HostingCdnOrderService.prepareOrderCart(user.ovhSubsidiary)
-        .then((cartId) => {
-          data.cartId = cartId;
-          return HostingCdnOrderService.addItemToCart(
-            cartId,
-            serviceName,
-            serviceOption,
-          );
-        })
-        .then((cart) => {
-          return { cart, cartId: data.cartId };
-        })
-        .catch((error) => goBackWithError(get(error, 'data.message', error)));
+    onSuccess: /* @ngInject */ ($translate, goBack) => (checkout) => {
+      const message = checkout.autoPayWithPreferredPaymentMethod
+        ? $translate.instant('hosting_dashboard_cdn_order_success_activation')
+        : $translate.instant('hosting_dashboard_cdn_v2_order_success', {
+            t0: checkout.url,
```

Comment:
Already existing translation ? If not maybe we could replace `t0` with something more descriptive like `url`

Reply:
Yes, it is a translation which already exists: https://github.com/ovh/manager/blob/aca48f5552088fe64d1dfcdb645932b34dac2182/packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json#L228